### PR TITLE
suggested updates to README

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+At some point it will also include a way to run this via maven, and possibly
+gradle, however, at this point in time it is CLI only.
 
-== Usage
+== Building and Running
 
-Build using `mvnw install -DskipTests` (until it's in a repo) then run the uberjar in `target` named `asciidoc-splitter-1.0-SNAPSHOT.jar` by using the java command:
-`java -jar target/asciidoc-splitter-1.0-SNAPSHOT.jar`.
-You will need to specify two options:
+Build using `./mvnw install -DskipTests` (until it's in a repo). This creates an
```
Comment (oraNod) [inline README.adoc]: @LightGuard hi Jason. this is a cool utility. I've just taken it for a quick test drive. tbh I just dove straight into the readme without looking at the contents of the repo. when I first ran I used "mvn" thinking "mvnw" was a typo. it did work but returned some WARN messages.

perhaps the above change might help others avoid a similar fate. cheers.

File: README.adoc [hunk 2]
```diff
@@ -1,15 +1,21 @@
 = Asciidoc Splitter
 
-A Java utility to split asciidoc files into a modular documentation layout.
-At some point it will also include a way to run this via maven, and possibly gradle, however, at this point in time it is CLI only.
+Java utility that split asciidoc files into a modular documentation layout.
+At some point it will also include a way to run this via maven, and possibly
+gradle, however, at this point in time it is CLI only.
```
Comment (LightGuard) [inline README.adoc]: I like this the way it was before.

File: README.adoc (hunk 1 above)
Comment (LightGuard) [reply README.adoc]: Fair enough

**Comment by LightGuard** [comment]
Thanks!

**Pull request merged**